### PR TITLE
Update CI testing data to include methyl matrices and gatk cn subsets

### PR DESCRIPTION
**Pull request by ewafula**
#### What scientific question is your analysis addressing?
Update [create-subset-files](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/create-subset-files) module to create methylation subset beta-values(`methyl-beta-values.rds`), m-values (`methyl-m-values.rds`), cn-values (`methyl-cn-values.rds`), and cnv-gatk(`cnv-gatk.seg.gz`) for CI testing data

#### What was your approach?
1). Created a script to select a list of sample IDs from all three methylation matrices to create a CI testing subset datasets
2). Using the histologies file and independent samples lists, randomly selected a list of 5 methylation sample IDs from 850K (CBTN) and 450K (TARGET) arrays each and corresponding RNA-Seq samples for patients who have both datasets
3). Included the selected 10 methylation sample IDs in the subset data sets to enrich for methylation samples
4). Included the selected 10 RNA-Seq sample IDs in the subset data sets to enrich for RNA-Seq samples with methylation data
5). updated the `copy_number_consensus_call` and the `methylation-summary` modules to read input data files from the release `data/` directory
6). Uploaded the updated CI testing subset files to the s3 bucket - `s3://d3b-openaccess-us-east-1-prd-pbta/open-targets/testing/`

#### What GitHub issue does your pull request address?
- https://github.com/PediatricOpenTargets/ticket-tracker/issues/493

#### Is there anything that you want to discuss further?
NA

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
YES 

#### What types of results are included (e.g., table, figure)?
CI test subset data files

#### What is your summary of the results?
https://github.com/PediatricOpenTargets/ticket-tracker/issues/493

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+The histologies file contains the information we need for identifying samples for patients with both methylation and RNA-Seq data in a cancer_group-cohort combination.
+
+```{r}
+histologies_df <- read_tsv(file.path("..", "..", "data", "histologies.tsv")) %>% 
+ filter(sample_type == "Tumor" & 
+          (tumor_descriptor == "Primary Tumor" | 
+             tumor_descriptor == "Initial CNS Tumor") & 
+          (experimental_strategy == "RNA-Seq" | 
+             experimental_strategy == "Methylation") & 
+          (cohort == "TARGET" | cohort == "PBTA")) %>%
```
Comment (jharenza) [inline analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd]:
```suggestion
 filter(sample_type == "Tumor",
          tumor_descriptor %in% c("Primary Tumor", "Initial CNS Tumor"),
          experimental_strategy %in% c("RNA-Seq", "Methylation"),
          cohort %in% c("TARGET", "PBTA")
          ) %>% 
```

absolutely not a necessary update but a suggestion for more concise code :). Note: this is suggested but untested and spacing may be off due to github editing

File: analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  
+```
+### Methylation samples
+
+Get methyl biospecimen identifiers in methyl independent sample list
+
+```{r}
+independent_samples <- read_tsv(
+  file.path("..", "..", "data", 
+            "independent-specimens.methyl.primary.eachcohort.tsv"))
```
Comment (jharenza) [inline analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd]: I might prefer you set a `root_dir` and map the data folder filepath from there

File: analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+```{r}
+set.seed(2022)
+```
+
+The histologies file contains the information we need for identifying samples for patients with both methylation and RNA-Seq data in a cancer_group-cohort combination.
+
+```{r}
+histologies_df <- read_tsv(file.path("..", "..", "data", "histologies.tsv")) %>%
```
Comment (jharenza) [inline analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd]: I might prefer you set a `root_dir` and map the data folder filepath from there

File: analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  rename(Kids_First_Biospecimen_ID_Methyl = Kids_First_Biospecimen_ID)
+
+```
+
+### RNA-Seq samples 
+
+Get rnaseq biospecimen identifiers in rnaseq independent sample list
+
+```{r}
+independent_samples <- read_tsv(
```
Comment (jharenza) [inline analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd:67]: I might prefer you set a `root_dir` and map the data folder filepath from there

File: analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  file.path("..", "..", "data",
```
Comment (jharenza) [inline analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd]: I might prefer you set a `root_dir` and map the data folder filepath from there

File: analyses/create-subset-files/01-get_biospecimen_identifiers.R [hunk 6]
```diff
@@ -329,7 +335,7 @@ nf1_rnaseq <- c("BS_81SP2HX4", "BS_KFD5128N", "BS_YDEVMD24",
 
 #### Samples we need to include to run rnaseq-batch-correct module -------------
 
-# For more information, see the 00-enrich-positive-examples notebook
+# For more information, see the 00-enrich-batch-correction-examples.Rmd notebook
```
Comment (jharenza) [inline analyses/create-subset-files/01-get_biospecimen_identifiers.R:338]: Suggest adding the module name as well

**Comment by jharenza** [comment]
@ewafula I downloaded the testing files and all md5sums check out. Merging dev back in - could have been a wonky thing with GA, so let's see what happens with a rerun.

**Review by jharenza — COMMENTED** [review]
I realized I did not push these comments..

**Comment by jharenza** [comment]
CN modules are down to ~5 minutes 🎉 , but some other checks failed..

File: analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd (hunk 2 above)
Comment (ewafula) [reply analyses/create-subset-files/00-enrich-methyl-rnaseq-examples.Rmd]: Ok, I will update. It was previously done like this in original OpenPBTA module

**Review by jharenza — APPROVED** [review]
🚀

**Pull request merged**